Common/LogManager.cpp

```diff
@@ -51,8 +57,14 @@ void GenericLog(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type, const char
 		instance->Log(level, type, file, line, fmt, args);
 	} else {
 		// Fall back to printf if we're before the log manager has been initialized.
+#if PPSSPP_PLATFORM(ANDROID)
+		char temp[512];
+		vsnprintf(temp, sizeof(temp), fmt, args);
+		__android_log_print(ANDROID_LOG_INFO, "PPSSPP", "EARLY: %s", temp);
```

Comment:
__android_log_vprint ?

Reply:
Oh! didn't know about that one.

Reply:
Although, then I won't be able to prepend "EARLY: " which I kinda wanna do. These aren't performance critical anyway.